model/Individual/Individual.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Individual
+
+## Summary
+
+A profile to be used when sharing individual SPDX `Element`.
+
+## Description
+
+Although most use cases involve sharing `Elements` inside a `Collection`, there are cases where a stand-alone SPDX element may be made available.
+In those cases, the `Element` `creationInfo` property would be required to identify the `Entity` that created the element.
```

Comment:
`Entity` will be called `Agent` soon.

Reply:
Agree - I left it as the current definition to match the repo

Reply:
It has now been updated to Agent